hackathon_site/dashboard/frontend/src/components/general/ProductOverview/ProductOverview.test.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import SideSheetRight from "../SideSheetRight/SideSheetRight";
```

Comment:
For things that aren't actually being tested, use the absolute import `components/general/SideSheetRight/SideSheetRight`